src/Libraries/Microsoft.Extensions.AI.Abstractions/Tools/HostedMcpServerTool.cs

```diff
@@ -99,4 +99,9 @@ private static string ValidateUrl(Uri serverUrl)
     /// </para>
     /// </remarks>
     public HostedMcpServerToolApprovalMode? ApprovalMode { get; set; }
+
+    /// <summary>
+    /// Gets or sets additional headers to include in requests to the remote MCP server.
```

Comment:
From the previous version of this:
```suggestion
    /// Gets or sets the HTTP headers that the AI service should use when calling the remote MCP server.
```